inz-personal-data/Issues/ageLimits/description.xml

```diff
@@ -0,0 +1,48 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo INZ Personal Data Age Limits</en>
+        <nl>Demo INZ Personal Data Age Limits</nl>
+    </Name>
+    <ShortName>
+        <en>Age Limits</en>
+        <nl>Age Limits</nl>
+    </ShortName>
+    <SchemeManager>irma-demo</SchemeManager>
+    <IssuerID>inz-personal-data</IssuerID>
+    <CredentialID>ageLimits</CredentialID>
+    <Description>
+        <en>Over 18</en>
+        <nl>Over 18</nl>
```

Comment:
This describes not this credential but one of the attributes in this credential.